src/resolvers-cassandra/Settings/queries.js

```diff
@@ -187,6 +187,32 @@ function termBlacklist(args, res) { // eslint-disable-line no-unused-vars
   });
 }
 
+function getTrustedSourcesRowKey(pipelineKey, externalSourceId, sourceType, rank) {
+  return pipelineKey + ',' + externalSourceId + ',' + sourceType +  ',' + rank;
+}
+
+function cassandraRowToTrustedSource(row) {
+  return {
+    rowKey: getTrustedSourcesRowKey(row.pipelinekey, row.externalsourceid, row.sourcetype, row.rank),
+    pipelineKey: row.pipelinekey,
+    externalSourceId: row.externalsourceid,
+    sourceType: row.sourcetype,
+    rank: row.rank
+  };
+}
+
+function trustedSources(args, res) { // eslint-disable-line no-unused-vars
```

Comment:
you can rebase from master to get the latest approach for querying `fortis.trustedsources`, but we should be filtering off `pipelinekeys` and avoiding an whole table query.